3_javascript/javascript1/week3/script3.js

```diff
@@ -0,0 +1,135 @@
+// Remove the item in names that is equal to nameToRemove
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
```

Comment:
Try to use **[Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)** for formatting your code automatically.